test/controllers/posts_controller_test.rb

```diff
@@ -13,4 +13,16 @@ def test_get_show_successful
     assert_response :success
     assert_equal assigns(:post), posts(:one)
   end
+
+  test "posts new" do
+    get :new
+    assert_response :success
+  end
+
+  test "posts create" do
+    assert_difference 'Post.count', 1 do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.